mdtraj/reporters/basereporter.py

```diff
@@ -62,6 +62,7 @@ def __init__(
         velocities=False,
         atomSubset=None,
         enforcePeriodicBox=None,
+        forces=False
```

Comment:
If no other reporters store forces, could this just be changed in the child class?